From 876380c8eb574b781dab6011c475e9f70c0034f7 Mon Sep 17 00:00:00 2001
From: Colin Geniet <colin.geniet@gmail.com>
Date: Thu, 15 Feb 2024 20:23:49 +0100
Subject: [PATCH] hwdb: Remove version check in CH Pro Pedals rule

CH Pedals are incorrectly reported as an accelerometer [1], because they
have no button.  This is fixed by a rule in 60-input-id.hwdb [2], but
the rule checks id/version="0100", while my pedals report id/version="0111".

So there are several versions of the pedals, presumably all affected
by the bug. Remove the version check in the rule to fix them all.

[1] https://bugs.freedesktop.org/show_bug.cgi?id=81889
[2] commit: 230ed4c4ba (hwdb: CH Pro Pedals not classified correctly due to no buttons, 2022-01-19)
    PR: https://github.com/systemd/systemd/pull/22184

(cherry picked from commit bd083fc2ebcf4940cc750d8d4f39a1b5b7c9e154)

Related: RHEL-5950
---
 hwdb.d/60-input-id.hwdb | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/hwdb.d/60-input-id.hwdb b/hwdb.d/60-input-id.hwdb
index 1c84777fe0..eff160cd31 100644
--- a/hwdb.d/60-input-id.hwdb
+++ b/hwdb.d/60-input-id.hwdb
@@ -72,7 +72,7 @@ id-input:modalias:input:b0003v046Dp4066e0111*
  ID_INPUT_MOUSE=0
 
 # CH Products Pro Pedals
-id-input:modalias:input:b0003v068Ep00F2e0100*
+id-input:modalias:input:b0003v068Ep00F2*
  ID_INPUT_ACCELEROMETER=0
  ID_INPUT_JOYSTICK=1
 
